src/Components/Facility/TriageForm.tsx

```diff
@@ -263,54 +251,29 @@ export const TriageForm = (props: triageFormProps) => {
           },
         }}
         backUrl={`/facility/${facilityId}`}
-      />
-
-      <Modal
-        open={openModalForExistingTriage}
-        aria-labelledby="Triage Check"
-        aria-describedby=""
-        className=""
       >
-        <div className="h-screen w-full absolute flex items-center justify-center bg-modal">
-          <div className="bg-white rounded shadow p-8 m-4 max-h-full text-center flex flex-col max-w-lg w-2/3 min-w-max-content">
-            <div className="mb-4">
-              <i className="fa-solid fa-triangle-exclamation text-red-500 fa-4x"></i>
-              <h1 className="sm:text-xl text-sm">
+        <ConfirmDialogV2
+          title={
+            <div className="">
+              <i className="fa-solid fa-triangle-exclamation text-red-500 fa-3x"></i>
```

Comment:
I hope you've read the EPIC issue of material UI. Use <CareIcons /> instead of font-awesome icons in upgraded pieces. Also the size is too big, we could use `text-lg`.

Refer the epic issue to know more on how to use care icons exactly.

```suggestion
              <CareIcon className="care-l-exclamation-triangle text-red-500 text-lg" />
```